tests/test_tap_kafka.py

```diff
@@ -0,0 +1,118 @@
+import pytest
```

Comment:
I think would be also really nice to check that tap produces all the messages as expected for some fake config :)

Reply:
makes sense, I'm going to update the PR

Reply:
addressed and added tests by https://github.com/transferwise/pipelinewise-tap-kafka/pull/5/commits/c19873e67d3fff22f3b4a11b435bc53609ba9064